src/phoenix/session/client.py

```diff
@@ -117,6 +119,37 @@ def query_spans(
             df = results[0]
             return None if df.shape == (0, 0) else df
         return results
+        if len(results) == 1:
+            df = results[0]
+            return None if df.shape == (0, 0) else df
+        return results
+
+    def get_evaluations(self) -> List[Evaluations]:
+        if self._use_active_session_if_available and (session := px.active_session()):
+            return session.get_evaluations()
+        response = self._session.post(f"{self._base_url}/v1/get_evaluations")
```

Comment:
Would we consider switching to an async requestor backend? Maybe we could avoid using `run_in_executor` eventually

Reply:
That's a good point. However, because these are cpu-bound, the executor needs to be called at some point, so the benefit of an async backend is largely ceremonial.